src/index.ts

```diff
@@ -103,16 +103,19 @@ export const service: KapShareService = {
       title: 'Storage Account Name',
       type: 'string',
       default: '',
+      required: true,
     },
     accountKey: {
       title: 'Storage Account Key',
       type: 'string',
       default: '',
+      required: true,
```

Comment:
Do any of these fields have a minimum or maximum length? Or do they have to follow some kind of pattern. Would be nice to add some validation to them. For example, `minLength` or a regex `pattern`.

https://json-schema.org/understanding-json-schema/reference/string.html#regular-expressions

Reply:
The account key is a [base64](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator#authorize-with-shared-key-credentials) string. However, everyone will just be copying and pasting that from the Azure Portal, so I'm not sure additional validation would add much value and I'm not sure if there's esoteric cases that could generate other keys.

For the storage account name, the Azure portal says:

> It must be 3 to 24 characters long, and can contain only lowercase letters and numbers.

I think adding validation for that would be good, particularly so that people make sure to enter `<accountName>` instead of `<accountName>.blob.core.windows.net`

Reply:
> The account key is a base64 string. However, everyone will just be copying and pasting that from the Azure Portal, so I'm not sure additional validation would add much value and I'm not sure if there's esoteric cases that could generate other keys.

Good point. I just added a validation to ensure it's not empty.